graphql/src/main/java/com/scalar/db/graphql/datafetcher/QueryGetDataFetcher.java

```diff
@@ -69,6 +70,7 @@ Get createGet(Map<String, Object> getInput) {
     if (consistency != null) {
       get.withConsistency(Consistency.valueOf(consistency));
     }
+    helper.addProjections(get, environment);
```

Comment:
I'm not sure how this class and `DataFetcherHelper` are used but I have some thoughts about the class design.

First of all, is it OK that this class is not thread-safe? Multiple threads calling `createGet` would cause some issues due to the `helper.addProjections`?
Second, from a user's perspective, I wouldn't expect `createGet` to have a side effect on the  `DataFetcherHelper` object, so I think the interface might need to be revised.

Can this class create `projections` and return it by a (new) method?

Reply:
Actually, the `helper.addProjections()` method doesn't have any side effects on the `helper` instance, and the method could have been static. I've just made a fix for that. Then I don't think there is a problem with thread safety.
But the design of `DataFetcherHelper` is getting not good. I made it to delegate common procedures for all data fetchers, but now the responsibilities for that class are not clear. I'm going to refactor the structures of data fetcher classes.